### DISCUSSION TITLE
Add an overview of currently active (live) users

### DISCUSSION THREAD

**Discussion by bigbasti** [umami-software/umami#2985] [Ideas]
I think an overview of the currently active users would be really informative.

When you click on 
![image](https://github.com/user-attachments/assets/a1f5305e-28c2-4150-aa46-03368ab34524)
on the top, you open an overview with a table similar to the one in "Sessions" where you see the users, their origin, device and so on.
Also interesting to see would be which page they're currently on and a link to their session.

so no rocket science here but would be really informative. :-)

BONUS: A time graph showing how many active users there were in the time frames.

Mockup:

![image](https://github.com/user-attachments/assets/853c8f08-70b6-4998-b343-fd09f361d54a)

What do you think?

**Comment by mzohrab** [comment]
Would be great to have this in shared analytics as well! There is no way to see live users in shared environment.
![image](https://github.com/user-attachments/assets/206646a2-bf5c-4a13-ab69-7affa52999e4)
